examples/fabric-sync/bridge/BUILD.gn

```diff
@@ -24,47 +24,23 @@ config("config") {
   ]
 }
 
-chip_data_model("fabric-bridge-zap") {
-  zap_file = "fabric-bridge.zap"
-  is_server = true
-}
-
-# This includes all the clusters that only exist on the dynamic endpoint.
-source_set("fabric-bridge-common") {
-  public_configs = [ ":config" ]
-
-  sources = [
-    "${chip_root}/src/app/clusters/ecosystem-information-server/ecosystem-information-server.cpp",
-    "${chip_root}/src/app/clusters/ecosystem-information-server/ecosystem-information-server.h",
-  ]
-
-  public_deps = [ ":fabric-bridge-zap" ]
-}
-
 source_set("fabric-bridge-lib") {
   public_configs = [ ":config" ]
 
   sources = [
-    "include/Bridge.h",
-    "include/BridgedAdministratorCommissioning.h",
-    "include/BridgedDevice.h",
-    "include/BridgedDeviceBasicInformationImpl.h",
-    "include/BridgedDeviceManager.h",
+    "Bridge.cpp",
+    "Bridge.h",
+    "CommissionerControlDelegate.cpp",
+    "CommissionerControlDelegate.h",
+    "FabricAdminDelegate.h",
+    "FabricBridge.cpp",
+    "FabricBridge.h",
     "include/CHIPProjectAppConfig.h",
-    "include/CommissionerControlDelegate.h",
-    "include/FabricAdminDelegate.h",
-    "include/FabricBridge.h",
-    "src/Bridge.cpp",
-    "src/BridgedAdministratorCommissioning.cpp",
-    "src/BridgedDevice.cpp",
-    "src/BridgedDeviceBasicInformationImpl.cpp",
-    "src/BridgedDeviceManager.cpp",
-    "src/CommissionerControlDelegate.cpp",
-    "src/FabricBridge.cpp",
   ]
 
   deps = [
-    "${chip_root}/examples/fabric-sync/bridge:fabric-bridge-common",
+    "${chip_root}/examples/fabric-bridge-app/fabric-bridge-common",
+    "${chip_root}/examples/fabric-bridge-app/fabric-bridge-common:fabric-bridge-lib",
```

Comment:
Now we have fabric-sync depending on the lib defined within another example app, shall we refactor this common lib out and have both example apps depends on it?